dandi/tests/test_metadata.py

```diff
@@ -105,7 +106,7 @@ def test_metadata2asset(schema_dir):
     # data.json(exclude_unset=True, exclude_none=True, indent=2)
     json_data = """{
   "identifier": "0b0a1a0b-e3ea-4cf6-be94-e02c830d54be",
-  "schemaVersion": "0.1.1",
+  "schemaVersion": null,
```

Comment:
I think it would be best to validate that it corresponds to current `DANDI_SCHEMA_VERSION` or do you think it is not worth that?

Reply:
I believe that's taken care of by the `validate_asset_json()` call.

Reply:
ok then let's proceed, thanks